basic_sender.go

```diff
@@ -336,7 +331,7 @@ func (b *buildlogger) Close() error {
 	return catcher.Resolve()
 }
 
-func (b *buildlogger) createNewLog(ts time.Time) error {
+func (b *buildlogger) createNewLog() error {
```

Comment:
does the service now construct its own time?

Reply:
yes